File: CS.NET/FrmMDI.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;

namespace Softgroup
{
	public partial class FrmMDI
	{


		internal FrmMDI()
		{
			InitializeComponent();
		}
		private void NewWindowToolStripMenuItem_Click(object sender, System.EventArgs e)
		{
			FrmStandard oChild = new FrmStandard();
			oChild.MdiParent = this;
			oChild.Show();
		}

		private void NewWindowBToolStripMenuItem_Click(object sender, System.EventArgs e)
		{
			FrmDataGrid oChild = new FrmDataGrid();
			oChild.MdiParent = this;
			oChild.Show();
		}

	}


} //end of root namespace

File: VB.NET/FrmMDI.vb
Imports System.Windows.Forms

Public Class FrmMDI


    Private Sub NewWindowToolStripMenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles NewWindowToolStripMenuItem.Click
        Dim oChild As New FrmStandard
        oChild.MdiParent = Me
        oChild.Show()
    End Sub

    Private Sub NewWindowBToolStripMenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles NewWindowBToolStripMenuItem.Click
        Dim oChild As New FrmDataGrid
        oChild.MdiParent = Me
        oChild.Show()
    End Sub

End Class

